public/theme.css

```
@@ -195,6 +195,7 @@
 	--docsearch-modal-shadow: 0 0 var(--theme-glow-blur) var(--theme-glow-diffuse), inset 0 0 0 1px var(--theme-glow-highlight),
 		inset 0 0 var(--theme-glow-blur) var(--theme-glow-diffuse);
 	--docsearch-container-background: hsla(var(--color-gray-10), 0.8);
+	--docsearch-hit-active-color: var(--theme-bg);
```

I'd suggest an even darker color so it has better contrast in greyscale and in smaller font sizes. What do you think? (Same color we use for the main text in the light theme)
```suggestion
	--docsearch-hit-active-color: hsla(var(--color-gray-10), 1);
```

---

Yep I agree @Yan-Thomas, looks better thanks!